tensorboard/spanner.py

```diff
@@ -0,0 +1,514 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I would maybe put this in a subdirectory. Like tensorboard.db_driver.cloud_spanner or something. Keep in mind that this repo has a canonical 1:1 mapping into the internal repo and we will be supporting Spanner Spanner too.